scripts/release/promote/s3_to_s3.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
This file is actually just a modified version of the old `wheel_to_s3.sh` script